### ISSUE TITLE
> Is there a way to put this app into app store?

### ISSUE THREAD

**Issue by TIGamingTV** [TheNoim/project-sitnu#21]
> Is there a way to put this app into app store? 

Read the readme under the section TestFlight

_Originally posted by @gamersi in https://github.com/TheNoim/project-sitnu/issues/20#issuecomment-1437999976_
            
What does „If you want to donate your login, then I can submit the app to the app store“ mean? I can send him my logins, because its my schools and i dont care. Is that what it mean?

**Comment by gamersi** [comment]
@TheNoim I think that's what's meant

**Comment by TheNoim** [comment]
Yes, kind of. There is another issue. Of course, for an app store page, I also need screenshots and a text. The issue with screenshots is that they obviously can't violate privacy. A few years ago, this wasn't an issue, because the demo untis server existed with fake data. 

I thought about creating a mock server, which returns fake untis data. This server I can also use for app store submission and as a bonus, I can use it in https://github.com/SchoolUtils/WebUntis for testing purposes. But I have a full time job and at the moment it is hard for me to allocate time for this. I have other side projects.

**Comment by TIGamingTV** [comment]
![24FB0EFB-FA5B-41A3-A69A-10DEAA444C48](https://user-images.githubusercontent.com/82331504/220766512-59705c79-234d-44a3-87f2-5fe16038be09.png)
![FAED5061-F7A3-4532-91BA-2E575DA23407](https://user-images.githubusercontent.com/82331504/220766517-045a0732-a2ae-4be7-8774-84f47f386204.png)
![1FA30F57-241E-417C-BBA1-302F7556F61E](https://user-images.githubusercontent.com/82331504/220766520-d43aead9-36cd-4af6-9cfc-270cf20fa90d.png)
![B21F5FA1-B0ED-4FA9-8AA1-88DA0B381FE1](https://user-images.githubusercontent.com/82331504/220766523-b0937af2-d924-4ddf-ba98-a0fd6cb8ea34.png)
![3ABFA36B-0937-4B32-97F1-5C628831E68D](https://user-images.githubusercontent.com/82331504/220766524-fd6f1f3d-3726-4791-9518-54270443f516.png)
![03B98204-43E4-49FF-94A2-8C7E44BD4B43](https://user-images.githubusercontent.com/82331504/220766526-02f2e82b-97ae-4726-ba96-2fe4958b0af3.png)
![547703F8-8F33-4D5E-8244-876DBA9ABC15](https://user-images.githubusercontent.com/82331504/220766528-6eab55c0-ecac-4181-b31d-75fa47fbbd69.png)

Idk if this might help you.

**Comment by TIGamingTV** [comment]
![986FB7C4-7D4E-4AA5-8F4A-D8F2AEDE60BD](https://user-images.githubusercontent.com/82331504/220766908-3511fe3a-bec2-4b72-a655-ad4392fdd5c3.png)
![8D95B28A-8B69-4018-A73A-05CAB65086C7](https://user-images.githubusercontent.com/82331504/220766917-78fdc75c-2a9b-40f3-9a4d-2ce473dd4b7b.png)

**Comment by gamersi** [comment]
Or you could maybe just hardcore the data for screenshots.
Or get in touch with Untis GmbH to create a new mock Server because I see that useful for many devs. But I could look into making a mock server in the coming days if you want to.

**Comment by bifaciall** [comment]
I guess I could give you my login: username: FSG{class}(e.g. FSG5c) and Password: „Klasse{class}“(same as „class“ as first one, e.g. Klasse5c). choose any class (5-10, a-c(sometimes until d))

**Comment by bifaciall** [comment]
oh and school: FSG Ludwigsburg
